mailbox/store/src/main/java/org/apache/james/mailbox/store/quota/StoreCurrentQuotaManager.java

```diff
@@ -20,13 +20,13 @@
 package org.apache.james.mailbox.store.quota;
 
 import org.apache.james.mailbox.exception.MailboxException;
-import org.apache.james.mailbox.model.QuotaRoot;
+import org.apache.james.mailbox.model.CurrentQuota;
 import org.apache.james.mailbox.quota.CurrentQuotaManager;
 
 public interface StoreCurrentQuotaManager extends CurrentQuotaManager {
 
-    void increase(QuotaRoot quotaRoot, long count, long size) throws MailboxException;
+    void increase(CurrentQuota currentQuota) throws MailboxException;
```

Comment:
Not sure with this if `CurrentQuota` is the right name for this pojo. It feels like "I increase my current quota with a current quota"... Is this ok? or someone has a better suggestion?